src/to_python.rs

```diff
@@ -927,4 +930,88 @@ private_impl_to_python_with_reference!(&self, py, UtcOffset => Py<PyTzInfo> {
 #[cfg(feature = "time")]
 private_impl_to_python_pyany!(UtcOffset => Py<PyTzInfo>);
 
+// ==== IndexMap =====
+
+#[cfg(feature = "indexmap")]
```

Comment:
minor thing - could you push these to another module & file so as to only need one `cfg`?

Reply:
refactored the indexmap and complex implementations to their own sub modules.